src/tools/rust-analyzer/crates/cfg/src/dnf.rs

```diff
@@ -200,7 +200,7 @@ impl Literal {
 impl fmt::Display for Literal {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         if self.negate {
-            write!(f, "not(")?;
+            f.write_str("not(")?;
```

Comment:
I think most subtree's will want to have these changes made in their respective repo, not here.

Reply:
I did not realize rust uses both submodules and git-subtree.